quickstart/public/index.css

```diff
@@ -1,120 +1,83 @@
-@import url(https://fonts.googleapis.com/css?family=Share+Tech+Mono);
-
-div#controls {
-  display: flex;
-  justify-content: center;
-  flex-direction: row;
+html {
+  height: 100%;
 }
 
-div#remote-media {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  height: 43%;
-  width: 100%;
-  background-color: #fff;
-  text-align: center;
-  margin: auto;
+body {
+  height: 100%;
 }
 
-div#remote-media video {
-  border: 1px solid #272726;
-  max-width: 80% !important;
-  background-repeat: no-repeat;
-  background-color: lightgrey !important;
-  background-image: url("https://static0.twilio.com/marketing/bundles/archetype/img/logo-wordmark.svg");
-  background-position: 50%;
+div.container-fluid {
+  height: 100%;
 }
 
-div#controls div#room-controls,
-div#controls div#preview div#log {
-  width: 16em;
-  margin: 0 1.5em;
-  text-align: center;
+div#participants {
+  overflow-y: auto;
 }
 
-div#controls div#preview button#button-preview {
-  display: flex;
-  justify-content: center;
-  align-items: center;
+div.participant {
+  background: center no-repeat url("data:image/svg+xml;utf8,<svg class='bi bi-person-fill' width='80' height='45' viewBox='0 0 16 16' fill='currentColor' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' d='M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 100-6 3 3 0 000 6z' clip-rule='evenodd'/></svg>");
+  border: 1px solid gray;
+  display: inline-flex;
+  height: 90px;
+  margin: 15px 10px;
+  overflow: hidden;
+  width: 160px;
```

Comment:
I think `max-width: 160px;` works a little better here. It keeps the participants from getting cut off at certain widths. Then `.participant.main` would need `max-width: initial;`. 

Before:
![image](https://user-images.githubusercontent.com/12685223/79277039-4e153c80-7e66-11ea-8832-899bbf12bffc.png)

After:
![image](https://user-images.githubusercontent.com/12685223/79277076-5e2d1c00-7e66-11ea-9914-58ae8a0969ab.png)

Reducing the horizontal margin at these widths would help with all that extra white space too.